joomla/joomla-cms
# Add css class option to image element of menu items

Pull Request for Issue #16447  .

### Summary of Changes
Currently we can set a custom class only for anchor element of a menu item.
This change add the option to set a custom class also for the image element associated to the menu item (if any). This way css sprites can be easily used to build menus with icons instead of using many single images, thus reducing overhead.


### Testing Instructions
Simply add a new menu element and notice the presence of "Image CSS style" option under the "Link options" tab.


### Expected result
Adding a class name under "Image CSS style" will apply a custom class to the img element of menu link item.


### Actual result
It's not possible to add a custom class to the img element.
